praw/reddit.py

```diff
@@ -296,19 +296,21 @@ def _prepare_objector(self):
                     'ModmailConversation': models.ModmailConversation,
                     'ModmailMessage': models.ModmailMessage,
                     'Submenu': models.Submenu,
+                    'TrophyList': models.TrophyList,
                     'UserList': models.RedditorList,
                     'button': models.ButtonWidget,
                     'calendar': models.Calendar,
                     'community-list': models.CommunityList,
                     'custom': models.CustomWidget,
                     'id-card': models.IDCard,
                     'image': models.ImageWidget,
+                    'menu': models.Menu,
                     'modaction': models.ModAction,
                     'moderators': models.ModeratorsWidget,
-                    'menu': models.Menu,
                     'more': models.MoreComments,
                     'stylesheet': models.Stylesheet,
                     'subreddit-rules': models.RulesWidget,
+                    't6': models.Trophy,
```

Comment:
Once this is added to kinds then can we use the dynamic config format like the ones used at the top of `mappings`?